cmd/tetra/debug/stats.go

```diff
@@ -0,0 +1,35 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright Authors of Tetragon
+
+package debug
+
+import (
+	"fmt"
+	"log"
+	"time"
+
+	"github.com/cilium/ebpf"
+	"github.com/spf13/cobra"
+	"golang.org/x/sys/unix"
+)
+
+func NewEnableStatsCmd() *cobra.Command {
+	cmd := cobra.Command{
+		Use:   "enable-stats",
+		Short: "Enable BPF stats",
+		Run: func(_ *cobra.Command, _ []string) {
+			// Enable bpf stats
+			stats, err := ebpf.EnableStats(uint32(unix.BPF_STATS_RUN_TIME))
+			if err != nil {
+				log.Fatal(fmt.Errorf("failed to enable stats: %v", err))
+			}
+			defer stats.Close()
```

Comment:
Could you explain how EnableStats works? It seems like it's a system-wide setting, and I'm guessing it's disabled when we call `.Close()`. If it's already enabled when we call EnableStats, will it remain enabled when we call Close?

Reply:
it enables stats system wide and returns file descriptor, when you close it the stats are disabled

it would be ok without the fd close because it will be closed when tetra app terminates,
but this it marks the window where stats are enabled

I'll add more comments with this explanation